DXMainClient/Domain/Multiplayer/TeamStartMapping.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using ClientCore;
+using Newtonsoft.Json;
+
+namespace DTAClient.Domain.Multiplayer
+{
+    public class TeamStartMapping
+    {
+        private const char LIST_SEPARATOR = ',';
+        
+        public const string NO_TEAM = "x";
+        public const string RANDOM_TEAM = "-";
+        public static readonly List<string> TEAMS = new List<string>() { NO_TEAM, RANDOM_TEAM }.Concat(ProgramConstants.TEAMS).ToList();
+
+        [JsonProperty("t")]
+        public string Team { get; set; }
+
+        [JsonProperty("s")]
+        public int Start { get; set; }
+
+        [JsonIgnore]
+        public bool IsValid => TeamIndex != -1;
+
+        [JsonIgnore]
+        public int TeamIndex => TEAMS.IndexOf(Team);
+
+        [JsonIgnore]
+        public int TeamId => ProgramConstants.TEAMS.IndexOf(Team);
+
+        [JsonIgnore]
+        public int StartingWaypoint => Start - 1;
+
+        [JsonIgnore]
+        public bool IsBlock => Team == NO_TEAM;
+
+        public override string ToString() => Team;
```

Comment:
The reason I don't think using ToString is good here is because it's `public` and is usually a representation of this object, which is pretty much conventional, so you would want an exhaustive (in a way; it may not be the full information but at least start and team should be enough) information about the object's contents. While team is surely a part of the object representation the full info would be team and start, so this override doesn't really describe the object exhaustively as it should. And because the method is so small I suggest just to remove it and just use a lambda.